### PR TITLE
docs: Clarify naming requirements for segments with multiple words

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

**Comment by jskeet** [comment]
The GH action failures appear to be unrelated to this PR.

File: proto/README.md [hunk 1]
```diff
@@ -76,6 +76,23 @@ events for multiple resources in the same product (e.g. buckets and
 objects within Cloud Storage) can use common messages in their
 definitions.
 
+The product and version parts must be all lower-case to conform with proto
+package naming conventions. A product may consist of multiple
+segments (e.g. `cloud.storage`) but if a single segment consists of
+multple words, they should be simply lower-cased. Example:
```
Comment (grant) [inline proto/README.md:82]: sp: multiple

File: proto/README.md [hunk 2]
```diff
@@ -76,6 +76,23 @@ events for multiple resources in the same product (e.g. buckets and
...
+`firebase.remoteconfig` for the Firebase Remote Config product.
```
Comment (grant) [inline proto/README.md:83]: nit:
"Example: `firebase.remoteconfig` for the Firebase Remote Config product." 
sounds a bit weird in the middle of this paragraph.

I'd imagine:
"For example, `firebase.remoteconfig` is the product name for Firebase Remote Config."

**Review by grant — APPROVED** [review]
2 requested changes. Approved.

CI fix is here: https://github.com/googleapis/google-cloudevents/pull/132

File: proto/README.md (hunk 1 above)
Comment (jskeet) [reply proto/README.md:82]: Done.

File: proto/README.md (hunk 2 above)
Comment (jskeet) [reply proto/README.md:83]: Done.

**Comment by jskeet** [comment]
Addressed comments and force-pushed an amended commit. Will merge when green-ish again.

**Pull request merged**